### ISSUE TITLE
[appveyor] Impossible to import anymore PyQt5

### ISSUE THREAD

**Issue by vallsv** [silx-kit/silx#1961]
```
PyQt5            5.10.1       
```
The version of PyQt5 did not change since February.

It could be a problem with Windows VC Redistribuable dependancy or something like that? Maybe not installed in latest Appveyor env. No idea.

Here is the output from the tests
```
python run_tests.py --installed -v --no-opencl --qt-binding %QT_BINDING%
INFO:silx.setup:Use setuptools
INFO:silx.setup:Use setuptools.setup
WARNING:run_tests:resource module missing
Traceback (most recent call last):
  File "run_tests.py", line 431, in <module>
    import PyQt5.QtCore  # noqa
ImportError: DLL load failed: The specified module could not be found.
```

**Comment by t20100** [comment]
It's related to numpy v1.15.0rc1. I reproduced locally, and see https://github.com/numpy/numpy/issues/11431.

**Comment by t20100** [comment]
closed by #1970
